docs/releases.md

```diff
@@ -0,0 +1,127 @@
+
+# Release Documentation
+
+# Abstract
+
+In order to provide a generic guidance to users, downstream maintainers and
+any other consumers of the Cloud Hypervisor project, this document
+describes the release process, release cadence, stability expectations and
+related topics.
+
+# Basic Terms
+
+## Stability
+
+In the context of Cloud Hypervisor, the points of reference regarding stability
+are as below:
+
+- [REST API](api.md#rest-api).
+- [Command line options](api.md#command-line-interface).
+- [Device Model](device_model.md).
+- Device tree, device list, ACPI, Hyper-V enlightenments and any other features
+  exposed to guest.
+- KVM compatibility.
+- UEFI compatibility.
+- Rust edition compatibility.
+
+This list might be by far incomplete. Still this document strives to gather any
+possible indicators to be desperately considered relevant for the stability context.
+
+## Experimental features
+
+Features that are being actively developed and are possibly incomplete or known to be
+unstable are considered experimental. These features are in no case recommended for
+use in production.
+
+Features currently considered experimental:
+
+- TDX.
```

Comment:
Lists should either have the punctuation of commas at the end (except for the last with a full stop) or no punctuation at all.

Reply:
Removed dots. Thanks